cookie.js

```diff
@@ -61,18 +53,18 @@ const fieldContentRegExp = /^[\u0009\u0020-\u007e\u0080-\u00ff]+$/ // eslint-dis
  * The object has the various cookies as keys(names) => values
  *
  * @param {string} str
- * @param {object} [options]
+ * @param {object} opt
```

Comment:
Why has this been changed? The code indicates it shouldn't be.

Reply:
Yeah you’re right, forgot to add the square brackets back

 I changed both function arguments to opt because changing them to options would take renaming more variables

It was just so that they would follow the same naming conventions

Reply:
I don't understand. It looks like the parameter names `options` were changed to `opt` without any other change?

Reply:
So `serialize` [had this line](https://github.com/fastify/fastify-cookie/pull/264/files#diff-57263c42de10223876c477ef63af8e65230cc0c5a629637daf6516ad093e3d1eL129) that simply assigned `options` or an empty object to `opt`, I removed that line because not creating an object will be faster and more memory efficient

Now, I needed to either change all the references of `opt` to `options`, the parameter name, or just change the parameter's name to `opt`

I changed it to `opt` and to have consistency among functions I also changed the other function's option parameter to opt, I can revert it if you'd like